app/client/components/transferAccount/transferAccountList.jsx

```diff
@@ -175,10 +175,10 @@ class TransferAccountList extends React.Component {
     if (this.state.account_type !== 'ALL') {
       // a vendor/recipient filter is applied
       if (this.state.account_type === 'VENDORS') {
-        filteredData = filteredData.filter(account => (account.is_vendor === true))
+        filteredData = filteredData.filter(account => account.is_vendor)
       }
       if (this.state.account_type === window.BENEFICIARY_TERM_PLURAL) {
-        filteredData = filteredData.filter(account => (account.is_beneficiary === true))
+        filteredData = filteredData.filter(account => !account.is_vendor)
```

Comment:
this is the fix for users not showing up correctly on accounts page. have we removed `is_beneficiary`? didn't see it in the code

Reply:
It shouldn't have been removed *yet*, though i don't love that the beneficiary type is defined on both the user and on the transfer_account, and they both have slightly different meanings.  This looks ok for now, and then we can work out a proper treatment later